.editorconfig

```diff
@@ -0,0 +1,15 @@
+[*]
+indent_style = space
+
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.html]
+indent_size = 4
+
+[*.js]
+indent_size = 2
+
+[*.scss]
+indent_style = tab
```

Comment:
Any reason to indent this with tabs over spaces?

Reply:
It's currently already tabs. I wanted to leave that alone rather than rewrite the whole file.